packages/@tinacms/graphql/src/resolver/index.ts

```diff
@@ -623,9 +623,16 @@ export class Resolver {
     assertShape<{ [key: string]: unknown }>(rawData, (yup) => yup.object())
     const value = rawData[field.name]
     switch (field.type) {
+      case 'datetime':
+        // See you in March ;)
```

Comment:
😂

Reply:
Made a joke about how it seems, every three months, we have to discuss dates 😆